Mathlib/SetTheory/Ordinal/Topology.lean

```diff
@@ -234,4 +234,126 @@ theorem enumOrd_isNormal_iff_isClosed (hs : s.Unbounded (· < ·)) :
     exact le_bsup.{u, u} _ _ (ha.2 _ hba)
 #align ordinal.enum_ord_is_normal_iff_is_closed Ordinal.enumOrd_isNormal_iff_isClosed
 
+open Set Filter
+
+/-- A positive ordinal is an accumulation point of a set of ordinals if there
+are elements in the set arbitrarily close to the ordinal from below. -/
+def IsAcc (o : Ordinal) (S : Set Ordinal) : Prop :=
+  o ≠ 0 ∧ ∀ p < o, ∃ s ∈ S, p < s ∧ s < o
+
+/-- A set of ordinals is closed in an ordinal if it contains all of
```

Comment:
I don't like the use of "in an ordinal". This phrasing makes sense in pen-and-paper mathematics where you treat ordinals as sets, but that's not quite what we're doing here. Maybe "below an ordinal" instead?